runtime/moonbase/src/lib.rs

```diff
@@ -1627,6 +1627,7 @@ moonbeam_runtime_common::impl_runtime_apis_plus_common! {
 	}
 }
 
+#[allow(dead_code)]
```

Comment:
How is this dead code?

This `CheckInherents` is very important.

Reply:
I think the reason is that it is never constructed (but it is used as an associated type). Also tanssi's PR marks it as dead code.